services/checks/checks_test.go

```diff
@@ -44,22 +45,27 @@ const (
 )
 
 func TestDownloadChecks(t *testing.T) {
-	t.Run("normal", func(t *testing.T) {
-		s, err := New(nil, nil, nil)
-		require.NoError(t, err)
-		s.host = devChecksHost
-		s.publicKeys = []string{devChecksPublicKey}
+	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+	db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
+
+	defer func() {
+		require.NoError(t, sqlDB.Close())
```

Comment:
We should not copy&paste that line in new tests because `testdb.Open` now users `t.Clean` to close database connection

Reply:
Removed